Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.cs

```diff
@@ -21,6 +22,9 @@ public partial class FirstPersonPlayer : FirstPersonCharacter
         private PlayerSlowMotionBehaviour _slowMotionBehaviour = null;
 
         [SerializeField] private bool _isThirdPerson = false;
+        
+        public event Action Dashed;
+        public event Action Slid;
```

Comment:
can u rename these as `OnDash`
`OnSlide`

Reply:
So with these I was matching the other actions in ecm2, their conventions are entirely just past tense forms, eg. `Jumped`, `Crouched`, etc